### ISSUE TITLE
Span problem and Stack field problem.

### ISSUE THREAD

**Issue by philipbaginski** [dcasia/expandable-table-row#10]
Trying to use span option and what I see: If span is set up as 1, all works fine, but starts from 2 there is a problem on the right side of the table.

![Screen Shot 2023-10-19 at 08 10 34](https://github.com/dcasia/expandable-table-row/assets/59031869/24193a69-77e6-4f26-aba2-7b156c8c56c8)

Also I have a Stack fields and looks like for this type of field package does not work.

**Comment by milewski** [comment]
> Trying to use span option and what I see: If span is set up as 1, all works fine, but starts from 2 there is a problem on the right side of the table.

You need to ensure that there are two columns available ahead of the column you set as "span2." For example, if you set "span: 1" and it's the last column in the table, there won't be enough space for it to span. As a result, you'll end up with an extra column, which is what you see in the screenshot you shared.

> Also I have a Stack fields and looks like for this type of field package does not work.

You can return an array and it will automatically wrapped in an stack field

**Comment by philipbaginski** [comment]
I have 15 columns and extended column is second one with span set up as 10. But any number above 1 is making the same result on the right side. Higher number - bigger space on the right side.

Regarding Stack field - can you show any sample how to use it?

**Comment by milewski** [comment]
https://github.com/dcasia/expandable-table-row/blob/a90ab5e1c1f8ce10b5f5ac260ed6913c5acd6b5a/src/ExpandableTableRowServiceProvider.php#L38

whatever value you return is already being wrapped into a stack field... so all you need to do is return an array like the example on the documentation:

```php
 Text::make('First Name')->expandableRowData(function () {
    return [
        Line::make(null)->displayUsing(fn () => 'Name')->asSubTitle(),
        Text::make('Full Name', fn (User $user) => sprintf('%s %s',$user->first_name, $user->last_name))->copyable(),
        Text::make('Email')->copyable(),
    ];
}),
```

**Comment by philipbaginski** [comment]
It works. But I have Stack field with two fields inside. And this Stack field I would like to extend because of the content I want to show it should start in this place of index table.

https://nova.laravel.com/docs/resources/fields.html#stack-field

**Comment by milewski** [comment]
Fixed here: [v0.1.1](https://github.com/dcasia/expandable-table-row/releases/tag/v0.1.1)

**Comment by philipbaginski** [comment]
Thank you. Span works great. What about Stack field? 
![stack](https://github.com/dcasia/expandable-table-row/assets/59031869/ae17cf7f-b44f-49d4-bf4d-2b6d02bd56d8)

Using this approach Stack field is not extended.
